src/test/java/integration/harness/BasicBranchProjectFactory.java

```diff
@@ -47,6 +47,11 @@ public BasicBranchProjectFactory() {
     public FreeStyleProject newInstance(Branch branch) {
         FreeStyleProject job = new FreeStyleProject(getOwner(), branch.getEncodedName());
         setBranch(job, branch);
+        try {
+            job.setQuietPeriod(0);
```

Comment:
For test stability we maintain the previous quiet period of `0`.  But live behavior will change.